.vscode/launch.json

```diff
@@ -13,10 +13,17 @@
     {
       "type": "node",
       "request": "launch",
-      "name": "Debugger",
+      "name": "Launch cluster.js in debugger",
       "program": "cluster.js",
       "envFile": "${workspaceFolder}/.env"
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Launch server.js in debugger",
```

Comment:
This will start the server without kicking off Heroku.  You will need to do an "ng build" yourself before starting this if you've made client-side changes.